templates/.phpcs.xml.dist

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+    <description>Generally-applicable sniffs for WordPress plugins.</description>
+
+    <!-- What to scan -->
+    <file>.</file>
+    <exclude-pattern>vendor/</exclude-pattern>
+
+    <!-- How to scan -->
+    <arg value="sp"/> <!-- Show sniff and progress -->
+    <arg name="basepath" value="."/><!-- Strip the file paths down to the relevant bit -->
+    <arg name="colors" />
+    <arg name="extensions" value="php"/>
```

Comment:
Another potentially useful arg is `<arg name="parallel" value="8"/>` - the number can vary, but 8 is a nice starting point.

Reply:
:+1: Comment to add along-side it: `<!-- Enabled parallel processing when available for faster results. -->`